src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ViewCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ViewCommandParserTest {
+
+    private final ViewCommandParser parser = new ViewCommandParser();
```

Comment:
Could this initialisation be placed within a `@BeforeEach` `setUp` method?

Reply:
<img width="417" alt="image" src="https://github.com/AY2324S2-CS2103T-F14-3/tp/assets/122190177/e638b1ab-b39b-4e89-8bfe-7e2f0dbd6b96">
Would it be something like this?